src/Features/CSharp/Portable/MoveToNamespace/CSharpMoveToNamespaceService.cs

```diff
@@ -13,7 +13,8 @@ internal class CSharpMoveToNamespaceService :
         AbstractMoveToNamespaceService<NamespaceDeclarationSyntax, TypeDeclarationSyntax>
     {
         [ImportingConstructor]
-        public CSharpMoveToNamespaceService()
+        public CSharpMoveToNamespaceService(IMoveToNamespaceOptionsService optionsService)
```

Comment:
📝 This is exported as a workspace service, so it's not possible to import it directly like this. The likely solution is to export the implementation directly with `[Export(typeof(IMoveToNamespaceOptionsService))]` instead of exporting it as a workspace service.